src/components/Picker/BasePicker.js

```diff
@@ -164,6 +161,27 @@ function BasePicker(props) {
         setIsHighlighted(false);
     };
 
+    const {icon, size} = props;
+
+    const iconToRender = useMemo(
+        () =>
+            icon
+                ? () => icon(size)
+                : () => (
+                      <Icon
+                          src={Expensicons.DownArrow}
+                          // eslint-disable-next-line react/jsx-props-no-spreading
+                          {...(size === 'small'
+                              ? {
+                                    width: styles.pickerSmall().icon.width,
+                                    height: styles.pickerSmall().icon.height,
+                                }
+                              : {})}
+                      />
+                  ),
+        [icon, size, styles],
+    );
```

Comment:
I would suggest to use the `if` block here in order of better readability.

```suggestion
    const iconToRender = useMemo(() => {
        if (icon) {
            return () => icon(size);
        }

        return (
            <Icon
                src={Expensicons.DownArrow}
                // eslint-disable-next-line react/jsx-props-no-spreading
                {...(size === 'small'
                    ? {
                          width: styles.pickerSmall().icon.width,
                          height: styles.pickerSmall().icon.height,
                      }
                    : {})}
            />
        );
    }, [icon, size, styles]);
```